src/libraries/System.Text.RegularExpressions/src/System/Text/RegularExpressions/RegexParser.cs

```diff
@@ -1717,13 +1717,13 @@ private string ParseProperty()
         {
             if (CharsRight() < 3)
             {
-                throw MakeException(RegexParseError.IncompleteSlashP, SR.IncompleteSlashP);
+                throw MakeException(RegexParseError.InvalidUnicodePropertyEscape, SR.IncompleteSlashP);
```

Comment:
I think it is probably worth making the same update to the resource names that you've done to the enum since they did match before. It's OK to change resource ID"s.

Reply:
Ok, tx. Will do them.

Reply:
@danmosemsft, done, I've renamed all resource constants as well now.